### DISCUSSION TITLE
PSQ_Rheobase questions regarding new version

### DISCUSSION THREAD

**Discussion by t-b** [AllenInstitute/MIES#992] [Q&A] [answered]
From Tim H.:

> After updating MIES on one of our systems we encountered some errors regarding the PSQ_Rheobase analysis function. On MIES 2.1_20200723-652-g7de80d3C everything works fine, however in the newest version both the membrane voltage and injected current are going to really dangerous levels in the test pulse between sweeps (0--500 mV, the first sweep is fine). Only after switching back to VC and stopping  the auto bias function back in CC does the amplifier return to normal values. All other protocols work fine. There are no error messages except a warning that the autobias current is to large to compensate.I only tested with the model cell, I don't like to test this on real recordings. After switching back to the old MIES version everything is fine. The digitizer is a NI USB-6343. Do you have any idea what is  causing this error?

**Comment by t-b** [comment]
What is the new MIES version you tried which did not work? Can you post a pxp/nwb from both MIES versions?

**Comment by t-b** [comment]
Reply from Tim H.:

> It seems this problem is related to the NI digitizer. I tried the same 
> on a different system with a USB-6343 digitizer and could replicate the 
> error. With an ITC 18 everything is fine. I will send you link with pxp 
> files from both versions. In both recordings the test pulse is saved as 
> MIES can't record when the autobias is exceeding it's limit. The MIES 
> version is the most recent on Github, downloaded today.

**Comment by t-b** [comment, marked as answer]
This was fixed in https://github.com/AllenInstitute/MIES/pull/974 and https://github.com/AllenInstitute/MIES/pull/971.

**Reply by timheistek** [reply]
Hi Thomas

Today I tried MIES version 2.3_20210908-234-g796379d0 on a NI digitizer. I still get errors when running the PSQ_rheobase function using the model cell. The first sweep is fine, however after the first sweep the test pulse turns on between sweeps and the voltage starts shift. Only when I switch to VC and back to CC I get a normal testpulse. I never had this issue on a ITC18 digitizer.

Best,
Tim
